uw-frame-components/css/buckyless/general.less

```diff
@@ -36,6 +36,15 @@ button:focus {
   outline: none !important;
 }
 
+.aria-only {
```

Comment:
Aria by design does not affect rendering of the page.
However styles could be added that are based off aria attributes.